dev-tools/common.bash

```diff
@@ -42,8 +42,18 @@ setup_go_root() {
   local version=${1}
   export PROPERTIES_FILE=go_env.properties
   GO_VERSION="${version}" .ci/scripts/install-go.sh
+
+  # Setup GOROOT and add go to the PATH.
   # shellcheck disable=SC1090
   source "${PROPERTIES_FILE}" 2> /dev/null
+
+  # Setup GOPATH and add GOPATH/bin to the PATH.
+  if [ -d "${HOME}" ] ; then
+    setup_go_path "${HOME}"
+  else
+    setup_go_path "${GOROOT}"
+  fi
```

Comment:
In case the HOME is unset let's fall back to the GOROOT folder, I assume this won't happen in the `internal-ci` since the HOME=JENKINS_HOME

![image](https://user-images.githubusercontent.com/2871786/107576093-9b7a4b80-6be8-11eb-946d-688cfce68cda.png)

But wanted to play safe with a fallback plan since jobs are configured as Freestyle and no idea if there will be any case that the ENV could be misconfigured.